### PR TITLE
Added #83 Feature: add user profile component

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Please review and approve it.

### CODE REVIEW

File: src/store/Modules/Auth.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  email: string = "rshaw@aapnainfotech.com"
+  get user() {
+    return this.username
+  }
+  get mail() {
+    return this.email
+  }
+
+  @Mutation
+  SET_ALIAS_NAME(username: string) {
```
Comment (ankitvats1) [inline src/store/Modules/Auth.ts:22]: Try to maintain the same naming convention already used in the code. (camelCase)

File: src/store/Modules/Auth.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    this.username = username
+  }
+
+  @Action
+  SetAlias(username: string) {
```
Comment (ankitvats1) [inline src/store/Modules/Auth.ts:27]: Where are we using it?

Comment (Rahul139-stack) [reply src/store/Modules/Auth.ts:27]: yes, we can alias the username by action also, used for debugging purpose

**Review by wildone — CHANGES_REQUESTED** [review]
can you address akits comments please.

File: src/store/Modules/Auth.ts (hunk 1 above)
Comment (wildone) [reply src/store/Modules/Auth.ts:22]: @ankitvats1 when you do review comments select request change so that its clear on the PR.

Comment (ankitvats1) [reply src/store/Modules/Auth.ts:22]: Sure @wildone

**Review by wildone — APPROVED** [review]

**Comment by wildone** [comment]
@Rahul139-stack did you fix the conflict?

File: src/store/Modules/Auth.ts (hunk 1 above)
Comment (Rahul139-stack) [reply src/store/Modules/Auth.ts]: Resolved, It is all in camelCase format

**Review by ankitvats1 — APPROVED** [review]

**Pull request merged**